app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="row">
+  <div class="column">
+    <div class="card">
+      <h3>Total Users</h3>
+      <p>
+        <%= @total_users %>
+      </p>
+    </div>
+  </div>
+  <div class="column">
+    <div class="card">
+      <h3>Total Questions</h3>
+      <p>
+        <%= @total_questions %>
+      </p>
+    </div>
+  </div>
+  <div class="column">
+    <div class="card">
+      <h3>Total Answers</h3>
+      <p>
+        <%= @total_answers %>
+      </p>
+    </div>
+  </div>
+  <div class="column">
```

Comment:
Remove, this was not specified in the requirements.

Reply:
https://github.com/SpicyCookies/rails-interview-project/pull/7/commits/084522578eb9a1fb1566174844436b319aa5008a#diff-0f1de9b0b214d551c91c7cf0b6b545c2c2cb7f8155d28cbb5db35e63fedf9946